src/index.html

```diff
@@ -1,13 +1,347 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBIKE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400&family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
+              <img
+                src="./images/MyBiKE.png"
+                alt="logo"
+                class="logo__img"
+              >
+            </a>
+            <a href="/" class="icon icon--call header__call-icon"></a>
+            <a href="#menu" class="icon icon--menu-opener header__menu-opener"></a>
+          </div>
+          <h1 class="header__title">
+            Take the Streets
+          </h1>
+        </div>
+      </div>
+
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="/" class="logo">
+              <img
+                src="./images/MyBiKE.png"
+                alt="logo"
+                class="logo__img"
+              >
+            </a>
+            <a href="/" class="icon icon--cross"></a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__items">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__items">
+              <a href="#aboutUs" class="menu__link">About us</a>
+            </li>
+            <li class="menu__items">
+              <a href="#compareBikes" class="menu__link">Compare bikes</a>
+            </li>
+            <li class="menu__items">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__items">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a href="+1 234 5555-55-55" class="menu__call menu__call--phone">
```

Comment:
```suggestion
          <a href="tel:123455555555" class="menu__call menu__call--phone">
```
зараз переходить на неіснуючу сторінку